tasks/R_task.wdl

```diff
@@ -0,0 +1,21 @@
+task R_Task {
```

Comment:
This file and the R Dockerfile seems unrelated?

Reply:
i have removed from this PR